torchtune/models/qwen2/_model_builders.py

```diff
@@ -40,6 +40,52 @@ def qwen2_7b() -> Union[TransformerDecoder, TiedEmbeddingTransformerDecoder]:
     )
 
 
+def qwen2_0_5b() -> Union[TransformerDecoder, TiedEmbeddingTransformerDecoder]:
+    """
+    Builder for creating a Qwen2 model initialized w/ the default 0.5B parameter values
```

Comment:
Could we please have a small comment here pointing out that `tied_embeddings` is enabled by default for 0.5B and 1.5B models? : )